luceedebug/src/main/java/luceedebug/PrefixPathTransform.java

```diff
@@ -4,29 +4,29 @@
 
 class PrefixPathTransform implements ICfPathTransform {
     private final String idePrefix_;
-    private final String cfPrefix_;
-    public PrefixPathTransform(String idePrefix, String cfPrefix) {
+    private final String serverPrefix_;
+    public PrefixPathTransform(String idePrefix, String serverPrefix) {
         this.idePrefix_ = idePrefix;
-        this.cfPrefix_ = cfPrefix;
+        this.serverPrefix_ = serverPrefix;
     }
     public Optional<String> cfToIde(String s) {
```

Comment:
To take this all the way, can you rename the `ICfPathTransform` methods `cfToIde` and `ideToCf` to the now-in-vogue `serverToIde` and `ideToServer`?

Reply:
I'll squash everything down once it's reviewed but I made a second and third commit.

- The second implements your suggestion, above.
- The third renames `CfVm.java` to `LuceeVm.java`.  (This is on a separate commit to make it easier to veto.)

I left `CfEntityRef.java`'s name alone, since I think it's referring to CFML, rather than ColdFusion.

Reply:
Squashed.